commit e5b40c8e2c0aefc909683758f736f1ac79c7fa8a (refs/changes/84/284/2)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2016-08-12T17:29:07+00:00 (3 years, 2 months ago)
    
    WORKFLOW-208 Update node-workflow dependency to latest on master
    
    Reviewed by: Pedro P. Candel <kusorbox@gmail.com>

diff --git a/package.json b/package.json
index 2533c44..3812c3f 100644
--- a/package.json
+++ b/package.json
@@ -10,7 +10,7 @@
   },
   "dependencies": {
     "wf-moray-backend": "git://github.com/joyent/node-workflow-moray-backend.git#376532",
-    "wf": "git://github.com/kusor/node-workflow.git#5cdc62",
+    "wf": "git://github.com/joyent/node-workflow.git#cac32eec0fa906adeba683f7f9436a9247740f11",
     "bunyan": "1.3.4",
     "effluent-logger": "git+https://github.com/joshwilsdon/effluent-logger.git#d662f161a07f94045ad2afb45442931511c40e51",
     "restify": "2.6.1",
